src/components/App/App.test.js

```diff
@@ -70,3 +70,26 @@ test('selecting correct answer shows correct result', async () => {
 
     expect(screen.getByText(/Correct/i)).toBeVisible();
 });
+
+test('clicking on continue in first round result redirects to second round', async () => {
+    render(<App />);
+
+    const startButton = screen.getByRole('button', { name: /start/i });
+    fireEvent.click(startButton);
+
+    const fiveRoundsButton = screen.getByRole('button', { name: '5' });
+    fireEvent.click(fiveRoundsButton);
+
+    await screen.findByText('1/5');
+    const correctOption = screen.getByRole('img', {
+        name: 'Juice WRLD',
+    });
+    fireEvent.click(correctOption);
+
+    await screen.findByText(/Correct/i);
+    const continueButton = screen.getByRole('button', { name: 'Continue' });
```

Comment:
really minor, but you can also use regular expressions that ignore casing in the name. It doesn't really matter to a user if they see "continue" or "Continue", so maybe it shouldn't matter to the test either. Depends a little bit on context and the system being tested though. Like most other rules, it's not an absolute rule!